src/query/storage/fanout/storage.go

```diff
@@ -42,11 +42,19 @@ type fanoutStorage struct {
 }
 
 // NewStorage creates a new fanout Storage instance.
```

Comment:
avoid having files in diffs with just ordering changes. This makes the diff long/complicated for no reason.

Reply:
There were changes to this previously with FetchRaw which got reverted